std/builtin/base.neva

```diff
@@ -74,4 +76,12 @@ pub component Join(data list<string>) (res string)
 
 // String
 #extern(split)
-pub component Split(data string) (res list<string>)
\ No newline at end of file
+pub component Split(data string) (res list<string>)
+
+#extern(upper)
+pub component ToUpper(data string) (res string)
```

Comment:
I guess these two are some kind of draft? Could you please remove them until you don't have actual implementation and working example?